EventFilter/RPCRawToDigi/plugins/RPCDigiMerger.h

```diff
@@ -0,0 +1,50 @@
+#ifndef EventFilter_RPCRawToDigi_RPCDigiMerger_h
+#define EventFilter_RPCRawToDigi_RPCDigiMerger_h
+
+#include <cstdint>
+#include <vector>
+#include <utility>
+#include <set>
+
+#include "FWCore/Framework/interface/ESHandle.h"
+#include "FWCore/Framework/interface/ESWatcher.h"
+#include "FWCore/Framework/interface/stream/EDProducer.h"
+
+#include "DataFormats/RPCDigi/interface/RPCDigi.h"
+#include "DataFormats/RPCDigi/interface/RPCDigiCollection.h"
+
+namespace edm {
+class ConfigurationDescriptions;
+class Event;
+class EventSetup;
+class ParameterSet;
+class Run;
+} // namespace edm
+
+class RPCDigiMerger
+    : public edm::stream::EDProducer<>
+{
+public:
+    RPCDigiMerger(edm::ParameterSet const & config);
+    ~RPCDigiMerger() override;
+
+    // static void compute_crc_64bit(std::uint16_t & crc, std::uint64_t const & word);
```

Comment:
Please remove the commented out line if not needed, or add another comment which explains why you want to keeo it